src/app/services/Challenges.js

```diff
@@ -44,10 +46,10 @@ function Challenges($http, SERVER) {
         })
     }
 
-    function getGivenChallenges(requestData) {
+    function getCompletedChallenges() {
         return $http({
             method: 'GET',
-            url: SERVER.ip + "/challenges/createdByStatus?" + requestData,
+            url: SERVER.ip + "/challenges/completedChallenges",
             withCredentials: true
```

Comment:
Could you not use `$httpProvider.defaults` in some sort of config to set `withCredentials` and, possibly, the domain of the `url` by default for the whole app, so you wouldn't need to do it for EVERY request?